airflow/www/views.py

```diff
@@ -679,62 +679,53 @@ def get_logs_with_metadata(self, session=None):
 
             return response
 
-        logger = logging.getLogger('airflow.task')
-        task_log_reader = conf.get('logging', 'task_log_reader')
-        handler = next((handler for handler in logger.handlers
-                        if handler.name == task_log_reader), None)
+        task_log_reader = TaskLogReader()
+        if not task_log_reader.is_supported:
+            return jsonify(
+                message="Task log handler does not support read logs.",
```

Comment:
Does Javascript need to be updated?

Reply:
Checking...

Reply:
Everything is fine with the logging on the website